esp/src/eclwatch/QuerySetDetailsWidget.js

```diff
@@ -262,6 +263,9 @@ define([
                 this.librariesUsedTab.set("tooltip", tooltip);
             } else if (name === "Clusters") {
                 if (lang.exists("ClusterQueryState.length", newValue)) {
+                    newValue.ClusterQueryState[0].MixedNodeStates ? dom.byId(this.id + "SuspendCluster").src = dojoConfig.getImageURL("mixwarn.png") : dom.byId(this.id + "SuspendCluster").src = dojoConfig.getImageURL("errwarn.png");
```

Comment:
Typically I only use the ```? :``` when assigning values to something ```var x = zzz ? qqq : aaa;``` 
Also are you sure that newValue.ClusterQueryState[0].MixedNodeStates will always exist?  or should you test if it exists first (I am not familiar with the data coming back so just don't know).